### PR TITLE
Require Node.js 14.14

### LINKED ISSUE #27: Improve install size
https://packagephobia.com/result?p=tempy

`tempy@0.7.0` install size is 1.61 MB

That's quite high. `0.6.0` was only 120KB.

### PR DESCRIPTION
**Pull request by Richienb**
Node.js 12 EOL in 1 week 🎉

Fixes #27

### CODE REVIEW

**Comment by sindresorhus** [comment]
The named export prefix should be `temporary`, not `tempy`.

File: index.d.ts [hunk 1]
```diff
@@ -41,132 +40,117 @@ The temporary path created by the function. Can be asynchronous.
...
+import {tempyWriteSync} from 'tempy';
+
+tempyWriteSync('🦄');
+//=> '/private/var/folders/3x/jf5977fn79jbglr7rk0tq4d00000gn/T/2f3d094aec2cb1b93bb0f4cffce5ebd6'
+```
+*/
+export function tempyWriteSync(fileContent: string | Buffer | TypedArray | DataView, options?: FileOptions): string;
 
-export default tempy;
+export {default as tempyRoot} from 'temp-dir';
```
Comment (sindresorhus) [inline index.d.ts:156]: `rootTemporaryDirectory`

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/tempy/pull/41#discussion_r851910420

**Comment by Richienb** [comment]
@sindresorhus

**Pull request merged**